src/page-modules/contact/ticket-control-and-fee/formMachine.ts

```diff
@@ -0,0 +1,142 @@
+  id: 'selectForm',
+  initial: 'start',
+  context: {
+    title: '',
+  },
+  states: {
+    start: {
+      on: {
+        'choose.feeComplaintForm': {
+          target: 'agreement1',
+          actions: 'setTitle',
+        },
+        'choose.refundForm': {
+          target: 'refundForm',
+          actions: 'setTitle',
+        },
+        'choose.feedbackForm': {
+          target: 'feedbackForm',
+          actions: 'setTitle',
+        },
+      },
+    },
+
+    agreement1: {
+      initial: 'idleAgreement1',
+      on: {
+        'choose.refundForm': {
+          target: 'refundForm',
+          actions: 'setTitle',
+        },
+        'choose.feedbackForm': {
+          target: 'feedbackForm',
+          actions: 'setTitle',
+        },
+      },
+      states: {
+        idleAgreement1: {
+          on: {
+            agree1: 'agreement2',
+          },
+        },
+
+        agreement2: {
+          initial: 'idleAgreement2',
+          on: {
+            disagree1: 'idleAgreement1',
```

Comment:
I'm thinking that we could be more explicit, for example by using `disagreeFirstAgreement` instead of `disagree1`